gateway/1.4/modules/ROOT/pages/_partials/forward-proxy-config.adoc

```diff
@@ -0,0 +1,164 @@
+//tag::configuration[]
+A forward proxy, such as Squid, enables you to route external HTTP Flex Gateway connections through a proxy connection.  
+
+For both Connected Mode and Local Mode, you configure a forward proxy using YAML configuration files.
+
+The following diagram demonstrates how an internal network communicates with an external network using a forward proxy. 
+
+image:forward-proxy-diagram.png[align=center]
+
+As in the diagram, when using a forward proxy, HTTP communication out of your internal network must travel through the forward proxy. 
+
+In the previous example, all HTTP API upstream communication must also pass through the forward proxy. For more information about how to enable some connections to communicate directly, without the forward proxy, see <<noproxy-parameter, noProxy Parameter>>.
```

Comment:
Suggestion:

In this example, HTTP communication out of your internal network must travel through the forward proxy. All HTTP API upstream communication must also pass through the forward proxy...

Since this text immediately follows the example, you can say "In this example" instead of "In the previous example."